src/OSPSuite.Core/Domain/ParameterIdentifications/OutputMappings.cs

```diff
@@ -46,6 +46,11 @@ public virtual IEnumerable<OutputMapping> OutputMappingsUsingDataRepository(Data
          return All.Where(x => x.UsesObservedData(dataRepository));
       }
 
+      public virtual IEnumerable<DataRepository> AllDataRepositoryMappedTo(string outputFullPath)
+      {
+         return All.Where(x => x.FullOutputPath == outputFullPath).Select(x => x.WeightedObservedData.ObservedData);
```

Comment:
Parameter name not consistnet here `FullOutputPath ` vs `outputFullPath`
note: this will be the name of the OUTPUT with the simulation. Is this what you need? I would add a comment to the method for intellisense: 
fullOutputPath: Full path of the output including simulation name or sthg like this

Reply:
done